frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -35,9 +50,9 @@ export const sessionsTableLogic = kea<sessionsTableLogicType<Moment, SessionType
         setNextOffset: (nextOffset: number | null) => ({ nextOffset }),
         fetchNextSessions: true,
         appendNewSessions: (sessions) => ({ sessions }),
-        dateChanged: (date: Moment | null) => ({ date }),
         previousDay: true,
         nextDay: true,
+        setFilters: (filters: Record<string, any>, selectedDate: Moment | null) => ({ filters, selectedDate }),
```

Comment:
Usage of `any` is generally discouraged within ts, but I guess to type this properly we'd need to port PropertyFilters.js and some other code as well as well?

Reply:
Would definitely be cool to have a completely typed filter! Indeed lots of "legacy" code to upgrade to make this work well. Until then `any` is probably fine here. There's also `unknown`, but that's a small difference.